### PR TITLE
fix: issues

### LINKED ISSUE #185: [PE-D] Slight grammar in UG intro
I think the intro is short and sweet. Perhaps this extra 'or' could be taken out?

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: joszx/ped#1

### LINKED ISSUE #186: [PE-D] UG for add a new recipe
Perhaps the usage portion of the command can specify that a quantity is required for the ingredients field? I think it would make the UG more comprehensive!

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: YiFei2/ped#2

### LINKED ISSUE #188: [PE-D] Greater than or equals to - Quick Start
Some users might not be familiar with programming/coding notations such as the greater than or equal to notation, maybe using words would be better in this case?

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: joszx/ped#2

### LINKED ISSUE #189: [PE-D] Command "delete -x 0" returns an invalid command format error
Running the command "delete -x 0" returns an invalid command format error even though 0 is considered a valid integer as it is a non-negative number as specified in the error message.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: DaneMarc/ped#2

### LINKED ISSUE #191: [PE-D] Command format example
Was slightly confused by the command example given </recipe name or -x </recipe number from the list>>

Was thinking if the square <> being nested suggests something else?

I think either having just </recipe name > or <-x recipe number from the list> will be clearer

(Please ignore the backslash, just having <text> causes it not to show up)

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: SethCKL/ped#1

### LINKED ISSUE #196: [PE-D] Slight misinformation in UG
I believe that a single-click is enough to display contents of a recipe! :D

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: YiFei2/ped#5

### LINKED ISSUE #202: [PE-D] Omission of -x
view </recipe number from the list>
should be view <-x recipe number from the list>

(ignore the / as it is just to make the text between <> show up)

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: SethCKL/ped#2

### LINKED ISSUE #205: [PE-D] Ambiguous Error Message
Tried rearranging the inputs but got this error. Not too sure which number it is referring to. The time, serving or the ingredients quantity?

In this case I tried further testing and it seems to be referring to the quantity (and also cause the error message is displaying a float). Perhaps a more accurate error message could reflect this case?

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: RyanCheungJF/ped#5

### LINKED ISSUE #207: [PE-D] Doccumentation for ingredients
In the UG, it is only stated 'ingredients of recipes separated by pipe'. I don't believe that there is a statement in other parts of UG stating that first 2 fields of ingredients are compulsory. (i.e Name + Amount) Perhaps putting it somewhere in the docs would be good for users to know what to type when adding recipes.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: RyanCheungJF/ped#6

### LINKED ISSUE #211: [PE-D] Slight misspell
Slight misspell with the word separated.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: joszx/ped#6

### PR DESCRIPTION
**Pull request by gigituang**
close #207 #202 #189 #182 #185 #188 #191 #196 #211 #218 #205 #192 #189 #183 #186 #222

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the recipe identified by the name used in the displayed list of recipes.\n\n"
             + "Parameters:\n1. name (must be a valid name, not case-sensitive)\n"
-            + "2. index (must be a valid index, no negative numbers)\n\n"
+            + "2. index (must be a valid, non-zero positive integer)\n\n"
```
Comment (yusufaine) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:25]: consistency with UG
```suggestion
            + "2. index (must be a valid, non-zero positive number)\n\n"
```

File: src/main/java/seedu/address/logic/commands/ViewCommand.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
             + ": Displays the details of a recipe identified by the name of "
             + "the recipe in the recipe list.\n\n"
             + "Parameters:\n1. name (must be a valid name, not case-sensitive)\n"
-            + "2. index (must be a valid non-negative index)\n\n"
+            + "2. index (must be a valid non-zero positive index)\n\n"
```
Comment (yusufaine) [inline src/main/java/seedu/address/logic/commands/ViewCommand.java:25]: consistency with UG
```suggestion
            + "2. index (must be a valid non-zero positive number)\n\n"
```

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 3]
```diff
@@ -112,7 +112,7 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      */
     void fillInnerParts() {
         String welcomeMessage = "Welcome to McKitchen!\n\n"
-                + "To begin, you can use the \"help\" command to access our the Command Summary in our User Guide.";
+                + "To begin, you can use the \"help\" command to access our Command Summary in our User Guide.";
```
Comment (yusufaine) [inline src/main/java/seedu/address/ui/MainWindow.java:115]: repetitive "our"
```suggestion
                + "To begin, you can use the \"help\" command to access the Command Summary in our User Guide.";
```

**Review by yusufaine — APPROVED** [review]
LGTM, left some suggestions that would be good to clarify/implement before merging 🔥 🌮 🍝

**Review by blaketingyu — APPROVED** [review]
LGTM!

**Pull request merged**